bin/utils.py

```diff
@@ -375,15 +375,20 @@ def upload_rsync(self, fr, to, flags = []):
   def upload(self, fr, to, type):
     self.copy(fr, "%s/%s/%s" % (self.location_root, type, to))    
 
-def maven_build_distribution(version):
+def maven_build_distribution(version, args):
```

Comment:
I think we should pass `assetRepo` and `assetBrand` directly here as this is a generic util method and should not be tightly coupled to argsparse.

Reply:
+1